src/main/java/com/segment/analytics/android/integrations/appboy/AppboyIntegration.java

```diff
@@ -300,32 +301,81 @@ public void track(TrackPayload track) {
       mLogger.verbose("This Install Attributed event is not in the proper format and cannot be"
           + " logged. The exception is %s.", exception);
     }
+
     if (properties == null || properties.size() == 0) {
       mLogger.verbose("Calling appboy.logCustomEvent for event %s with no properties.", event);
       mAppboy.logCustomEvent(event);
       return;
     }
+
+    if (event.equals("Order Completed") || properties.containsKey("revenue")) {
```

Comment:
Is the case of the `revenue` contemplated because of the Segment spec or something we have agreed with our D&I team? Or is there another reason? 🤔 

I'd like to know to check with the rest of marketing teams if that is something they would expect or not.

Reply:
It was just a behavior present in the original integration component. I don't think I have seen it in the spec itself, but I don't see any reason to remove it either.

Reply:
I should add, you should be able to configure this behavior from the Braze destination settings panel in Segment.

![Captura de pantalla 2020-04-15 a las 8 32 05](https://user-images.githubusercontent.com/643554/79305391-9d557e80-7ef3-11ea-873a-4479f95ff2f9.png)

It's just that the app SDKs just ignore that option is even there 🤷‍♂️